### PR TITLE
Feature/typescript server

### PR DESCRIPTION
**Pull request by DenesDanko**
Pls. review, and test like hell!

### CODE REVIEW

**Review by rigo101 — APPROVED** [review]

File: server/src/controllers/superuser.controller.ts [hunk 1]
```diff
@@ -93,17 +100,18 @@ module.exports.addMember = async (req, res, next) => {
     }
 };
 
-module.exports.deleteMember = async (req, res, next) => {
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const deleteMember = async (req: any, res: Response, next: NextFunction): Promise<void> => {
     log.info(`${req.user.email} is deleting ${req.body.remove}.`);
```
Comment (KristofNalesnyik) [inline server/src/controllers/superuser.controller.ts]: I would initialize an email variable for replacing all req.body.remove

File: server/src/controllers/user.controller.ts [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+        next(err);
+    });
+    form.on('end', () => {
+        log.info('Upload successful.');
+    });
+};
+
+type MemberLevel = Pick<IMember, '_id' | 'level' | 'email'>;
+
+const findMemberLevel = (membersLevel: Array<MemberLevel>, registeredUser: IRegisteredUser): string => {
```
Comment (KristofNalesnyik) [inline server/src/controllers/user.controller.ts]: It is enough passing registeredUser.email to this function. No need for the whole object

**Review by KristofNalesnyik — COMMENTED** [review]
Commented to refactor some parts.

**Comment by DenesDanko** [comment]
Modifications are done, based on comments, please test and approve!

**Comment by KristofNalesnyik** [comment]
I committed the eslint fix for config.ts file

**Review by KristofNalesnyik — APPROVED** [review]
Looks good. Manually tested for a couple of minutes.

**Pull request merged**